common/collision.h

```diff
@@ -146,6 +149,11 @@ class CollisionDetector {
         // Check for collision
         for (int i = 0; i < m_RobotMap.size().area(); i++) {
             if (m_ObjectsMap.data[i] & m_RobotMap.data[i]) {
+                // Calculate collision point
+                const auto coordPixel = div(i, m_RobotMap.cols);
+                firstCollisionPosition.x() = m_XLower + (coordPixel.rem * m_GridSize);
+                firstCollisionPosition.y() = m_YLower + (coordPixel.quot * m_GridSize);
```

Comment:
I made a vaguely similar addition in my robot control PR, except I returned which side of the object the robot had collided with. We def could do both, but perhaps I should make the last small changes to that branch and merge it first

Reply:
:+1:

Reply:
Shall we leave this for a subsequent PR?